open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/inmemory-repository-connector/pom.xml

```diff
@@ -19,6 +19,14 @@
         The in-memory repository connector provides a local repository that is entirely in memory.  Its is useful for
         testing/developing OMASs and demos.
     </description>
+    <dependencies>
```

Comment:
Can not have a dependency for an OMAS in a pluggable component for the repository services.